src/Tribe/Validate.php

```diff
@@ -239,7 +239,10 @@ public function slug() {
 
 			// esc_url_raw does the work of validating chars, but returns the checked string with a
 			// prepended URL protocol; so let's use strpos to match the values.
-			if ( false !== strpos( $maybe_valid_value, $this->value ) ) {
+			if (
+				! empty( $maybe_valid_value )
+				&& false !== strpos( $maybe_valid_value, $this->value )
+			) {
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 8fcce3f2e534edb09de7fefad3961b20</sub>